components/throttle.js

```diff
@@ -0,0 +1,93 @@
+// throttle.js
+
+var _ = require('underscore');
+var fs = require('fs');
+
+var idIndex = 0;
```

Comment:
I think it would be better if variables were local to the component instance and not global.